### PR TITLE
[AKS] Support create aks clusters enabling managed identity

### PR DESCRIPTION
**Pull request by norshtein**
**History Notes:**  
(Fill in the following template if multiple notes are needed, otherwise PR title will be used for history note.)

[Component Name 1] (BREAKING CHANGE:) (az command:) make some customer-facing change.  
[Component Name 2] (BREAKING CHANGE:) (az command:) make some customer-facing change.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [X] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by norshtein** [comment]
Close and reopen to rerun CI.

**Comment by norshtein** [comment]
@feiskyer  @yungezz  @zqingqing1  could you please have a review?

**Comment by norshtein** [comment]
@yonzhan @yungezz @Juliehzl Could someone help review this PR?

**Comment by yonzhan** [comment]
add to S166

File: src/azure-cli/azure/cli/command_modules/acs/_help.py [hunk 1]
```diff
@@ -331,6 +331,9 @@
   - name: --api-server-authorized-ip-ranges
     type: string
     short-summary: Comma seperated list of authorized apiserver IP ranges. Set to 0.0.0.0/32 to restrict apiserver traffic to node pools.
+  - name: --enable-managed-identity
+    type: bool
+    short-summary: Using a system assigned managed identity to manage cluster resource group.
```
Comment (yungezz) [inline src/azure-cli/azure/cli/command_modules/acs/_help.py]: is the managed identity for this specifc cluster for resource group?

Comment (norshtein) [reply src/azure-cli/azure/cli/command_modules/acs/_help.py]: Yes. The cluster will get a system assigned identity, the identity will be used by this cluster's kube-controller-manager to managed resources in this cluster's "MC_" resource group.

**Comment by norshtein** [comment]
@yungezz thanks for reviewing! Could you please merge the PR early so we can catch the release at 03/10?

**Comment by norshtein** [comment]
@arrownj thanks for reviewing. Test added. It is not allowed to enable managed identity for the cluster after the cluster is created, `--enable-managed-identity` can only serve as a creation time parameter.

**Pull request merged**